brave/common/extensions/url_bindings.cc

```diff
@@ -330,7 +330,8 @@ void URLBindings::Parse(
   GURL gurl(url_string);
   gin::Dictionary dict = gin::Dictionary::CreateEmpty(isolate);
   if (gurl.has_username())
-    dict.Set("auth", gurl.username() + (gurl.has_password() ? ":" + gurl.password() : ""));
+    dict.Set("auth", gurl.username() + (gurl.has_password()
```

Comment:
just saw this now- this just moves part of it onto the next line (since it went over 80 chars length, lint failed)